bobtemplates/plone/site_initialization/.mrbob.ini

```diff
@@ -0,0 +1,24 @@
+[questions]
+; subtemplate_warning.question = Please commit your changes, before using a sub-template! Continue anyway? [N/y]
```

Comment:
do we need these line, or can we clean it up?

Reply:
ok, if it's a sub template, it should be there, otherwise you risk overwriting files and have no way going back.

Reply:
Yes, I finally included them